test/tools/router_check/validation.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package envoy.RouterCheckToolSchema;
+
+import "envoy/api/v2/core/base.proto";
+import "validate/validate.proto";
+
+// [#protodoc-title: RouterCheckTool Validation]
+
+// The Validation Schema of the envoy router check test files.
+// The accepted input formats for the test are json and yaml.
+// The tool transparently converts json/yaml into this proto schema.
+message Validation {
+  // A collection of test cases.
+  repeated ValidationItem tests = 1;
```

Comment:
min items 1 here?

Reply:
@mattklein123 Do you think this is a hard requirement or nice to have? I think a test file with no tests is also fine. One doesn't have to completely delete the file when removing the tests for some reason.

Reply:
I don't feel strongly about it if you think there is use for no tests.